### PR TITLE
feat: add UI cluster config changes for CSP

### PR DESCRIPTION
**Pull request by petey**
Context:
========
Following https://github.com/screwdriver-cd/ui/pull/244 cluster admins will need to add a setting for loading avatars in the UI.

Objective:
----------
Update docs for UI configuration

### CODE REVIEW

**Review by minzcmu — APPROVED** [review]
👍 LGTM

**Review by d2lam — APPROVED** [review]
Let's merge this after the other PR is merged.

**Review by jithine — APPROVED** [review]

**Pull request merged**
